index.html

```diff
@@ -890,7 +890,7 @@ <h2>Presentations</h2>
 
         <pre class="example nohighlight" title="Basic structure of a presentation">
 {
-  "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+  "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
```

Comment:
I guess in the case of DID Auth via the Credential Handler API you would now either look for the DID in a `PublicKeyCredential` within `verifiableCredential` ... or would just look in the proof and use the owner of the key that provided the signature? Just making sure that this still works for use cases for which we have already written initial implementations.

Reply:
Yes, you could use either approach. You have to look in a different location and reason about the data differently, but the original use case is still supported AFAICT.